<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveWiget</class>
 <widget class="QWidget" name="WaveWiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>571</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,1,1">
    <item>
     <widget class="QPushButton" name="expansion">
      <property name="text">
       <string>区间展开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reduction">
      <property name="text">
       <string>还原</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="yE">
      <property name="text">
       <string>Y+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="yR">
      <property name="text">
       <string>Y-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="xE">
      <property name="text">
       <string>X+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="xR">
      <property name="text">
       <string>X-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LMove">
      <property name="text">
       <string>←</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RMove">
      <property name="text">
       <string>→</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>571</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="XcustomPlot" name="uPwt" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>111</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2" columnminimumwidth="0,0">
    <item row="0" column="0">
     <widget class="QLabel" name="labelU1">
      <property name="text">
       <string>t1:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="maximum">
       <double>99999999999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelU3">
      <property name="text">
       <string>t2:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="maximum">
       <double>9999999999999.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelU4">
      <property name="text">
       <string>瞬时值</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="insValue">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XcustomPlot</class>
   <extends>QWidget</extends>
   <header>xcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
